Gemfile

```diff
@@ -29,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'rubocop', require: false
```

Comment:
You are right.
Should i correct all of the Rubocop ofenses?.
There are several that we can modify. Like "Line too long" (should it be allways 80?) or "missing top-level class documentation comment", in that case, do i modify them? or should we stick to rubocop's suggestions strictly?